lib/core/src/Cardano/Wallet/Primitive/CoinSelection/MA/RoundRobin.hs

```diff
@@ -1219,6 +1436,16 @@ newtype AssetCount a = AssetCount
     { unAssetCount :: a }
     deriving (Eq, Show)
 
+--------------------------------------------------------------------------------
+-- Constants
+--------------------------------------------------------------------------------
+
+-- | The greatest token quantity that can be encoded within an output bundle of
+--   a transaction.
+--
+maxTokenQuantity :: TokenQuantity
+maxTokenQuantity = TokenQuantity $ fromIntegral (maxBound :: Word64)
```

Comment:
It's a bit of a mouthful, but renaming maxTokenQuantity -> maxTokenQuantityInTxOut will prevent misleading any readers.

Reply:
I agree this is a good idea. 👍🏻 

Fixed in 0cfb86f7ffc555d740944bbac506e2676a73f701